src/web/app/pages-instructor/instructor-session-base-page.component.ts

```diff
@@ -221,6 +222,29 @@ export abstract class InstructorSessionBasePageComponent {
         { courseid: model.feedbackSession.courseId, fsname: model.feedbackSession.feedbackSessionName });
   }
 
+  /**
+   * Downloads the result of a feedback session in csv.
+   */
+  downloadSessionResult(model: SessionsTableRowModel): void {
```

Comment:
for the frontend, is this the only place from which an instructor can download results? (the sessions table?)

The API has support for more params - but they are not used anywhere at the moment. Like missing responses / stats is true by default and there is no section name / detail.

Reply:
For now I think this is the only place where the instructor can download session results. I'm not sure if there are other places to download results in the previous version. But we can add the feature to configure download params in future. I believe that can be a separate issue.

Reply:
okay I understand!